example/gin/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/instana/go-sensor/example/gin
+
+go 1.11
+
+require (
+	github.com/gin-gonic/gin v1.7.2
+	github.com/instana/go-sensor v1.29.1-0.20210608074124-c12a7d59b2d4
+	github.com/instana/go-sensor/instrumentation/instagin v0.0.0-20210608075151-0d8105436b0c // indirect
```

Comment:
We'd probably need to merge and release https://github.com/instana/go-sensor/pull/234 before this one, right?

Reply:
Yes, it is a plan